src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+    }
+    public static String getDetails(String[] splitInput, int indexOfBy) {
+        String details = "";
+        for (int i = indexOfBy + 1; i < splitInput.length; i++) {
+            details += splitInput[i] + " ";
+        }
+        return details;
+    }
+
+    public static void printConfirmation(Task[] tasks, int taskCount) {
+        int numberOfTasks = taskCount + 1;
+        printSeparator();
+        System.out.println(" Got it, I've added this task: ");
+        System.out.println("   " + tasks[taskCount]);
+        System.out.println(" Now you have " + numberOfTasks + " task(s) in the list.");
+        printSeparator();
+    }
+
+
+    //Adds new task
+    public static void taskAdder(Task[] tasks, String description, int taskCount) {
+        Task newTask = new Task(description);
+        tasks[taskCount] = newTask;
+        printSeparator();
+        System.out.println(" added: " + description);
+        printSeparator();
+    }
+
+    //Sets isDone of Task object to be true
+    public static void markTaskAsDone(Task[] tasks, String description, int taskCount) {
+        //Determine index of task to be marked as done
+        String[] splitDescription = description.split(" ");
+        if(splitDescription.length == 1) {
+            System.out.println("Invalid command!");
+            printSeparator();
+            return;
+        }
+        int taskNumber = Integer.parseInt(splitDescription[1]) - 1;
+        printSeparator();
+        //Check if taskNumber is out of bounds
+        if(taskNumber < 0 || taskNumber > taskCount - 1) {
+            System.out.println("Invalid task number!");
+            printSeparator();
+            return;
+        }
+        //the taskNumber is valid
+        tasks[taskNumber].markAsDone();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("  [" + "\u2713" + "] " + tasks[taskNumber].description);
+        printSeparator();
+    }
+
+    //Prints separator line
+    public static void printSeparator() {
+        System.out.println("______________________________________________________________________________");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        String[] splitInput;
+        boolean isFinished = true;
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello from\n");
+        printSeparator();
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        printSeparator();
+
+        //Loops infinitely until user inputs "bye"
+        while(isFinished) {
```

Comment:
Consider initializing isFinished to false instead, and changing to while (!isFinished), to make it easier to understand.